src/customfunctions/functions.js

```diff
@@ -0,0 +1,5 @@
+import * as MyFunctions from "./sample";
+
+// To expose the functions to Excel, we need to set the
+//    CustomFunctionsMapping variable with the set of functions
+CustomFunctionMappings = MyFunctions;
```

Comment:
It makes it unclear if you assign everything exported to CustomFunctionMappings.